drf_link_navigation_pagination/__init__.py

```diff
@@ -32,6 +35,20 @@ class LinkNavigationPagination(LimitOffsetPagination):
         if hasattr(settings, "DRF_LNP_HEADER_NUMBER_OF_OVERLAP_PATHS")
         else "X-Drf-Number-Overlap-Paths"
     )
+    header_max_pagination_size = (
+        settings.DRF_MAX_PAGINATION_SIZE
+        if hasattr(settings, "DRF_MAX_PAGINATION_SIZE")
+        else "X-Drf-Max-Pagination-Size"
+    )
+
+    def paginate_queryset(self, queryset, request, view=None):
+        if request.headers.get(self.header_max_pagination_size) is not None:
+            # https://www.django-rest-framework.org/api-guide/pagination/#limitoffsetpagination
+            self.max_limit = int(request.headers.get(self.header_max_pagination_size))
```

Comment:
**question:** What will happen if the header is not given? Can we have a test for this case?

Reply:
I think [this test case](https://github.com/juntossomosmais/drf-link-navigation-pagination/blob/release/max-limit/tests/integration/test_init.py#L203) cover this. The pagination will follow the default behaviour, that is, no restriction on the limit parameter.